.github/stale.yml

```diff
@@ -13,9 +13,20 @@ exemptLabels:
 staleLabel: stale
 # Comment to post when marking as stale. Set to `false` to disable
 markComment: >
+  Thanks for your contribution!
+
   This issue has been automatically marked as stale because it has not had
-  recent activity. It will be closed if no further activity occurs. Thank you
-  for your contributions.
+  recent activity. Because the Atom team treats their issues
+  [as their backlog](https://en.wikipedia.org/wiki/Scrum_(software_development)#Product_backlog) stale issues
+  are closed. If you would like this issue to remain open:
```

Comment:
Maybe we can be more explicit and say `are closed after one year of inactivity`.

Reply:
I'd prefer to not have to update the text here every time that we decide to change the time windows 😕